res/values/options.xml

```diff
@@ -279,5 +282,16 @@
 
     <string name="opt_superSlider_title">Super Slider</string>
     <string name="opt_superSlider_summary">Perfect shape slider!</string>
-
+	
+    <string name="opt_changespeed_title">Speed Modify</string>
+    <string name="opt_forcear_title">Force AR</string>
+    <string name="opt_enablenc_whenspeedchange_title">Enable NC in speed modify</string>
+    <string name="opt_enablespeedchange_title">Enable speed modify</string>
+    <string name="opt_enableforcear_title">Enable force AR</string>
+	
+	<string name="opt_calculate_slider_path_in_game_start_title">Calculate sliderpaths in game start</string>
+	<string name="opt_calculate_slider_path_in_game_start_summary">Increase load time. Increase fps when play complex sliders.(such as some aspire map\'s slider)</string>
```

Comment:
Improve grammar:
"Increases performance in beatmaps where complex sliders are present, but increases beatmap load time"

Reply:
fixed